src/Blazored.LocalStorage/LocalStorageService.cs

```diff
@@ -26,7 +31,7 @@ public async Task SetItemAsync(string key, object data)
             if (e.Cancel)
                 return;
 
-            await _jSRuntime.InvokeAsync<object>("localStorage.setItem", key, JsonSerializer.Serialize(data));
+            await _jSRuntime.InvokeAsync<object>("localStorage.setItem", key, JsonSerializer.Serialize(data, this._jsonOptions));

```

Comment:
Could we drop the `this` keyword.
```suggestion
            await _jSRuntime.InvokeAsync<object>("localStorage.setItem", key, JsonSerializer.Serialize(data, _jsonOptions));
```